publication-commons/src/test/resources/features/UserAccessRights.feature

```diff
@@ -0,0 +1,110 @@
+Feature: User access rights
+
+
+  Background:
+    Given that there is a database "PUBLICATIONS" with publications
+    And that DynamoDBPublicationService exists
+    And the DynamoDBPublicationService has a READ method
+    And the READ method requires a user with non empty username
+    And the READ method requires a non empty publication ID
+    And the DynamoDBPublicationService has an UPDATE method
+    And the UPDATE method requires a user with non empty username
+    And the UPDATE method requires a non empty publication ID
+    And the DynamoDBPublicationService has an DELETE method
+    And the DELETE method requires a user with non empty username
+    And the DELETE method requires a non empty publication ID
+    And the DynamoDBPublicationService has a CREATE method
+    And the CREATE method requires a user with non empty username
+    And the CREATE method requires a non empty publication object
+    And the DynamoDBPublicationService has a PUBLISH method
+    And the PUBLISH method requires a user with non empty username
+    And the PUBLISH method requires a non empty publication ID
+    And there is a user with username "theUser"
+    And the user "theUser" is affiliated with the institution "theInstitution"
+    And the user's role is USER
+    And the user "theUser" does not have any other role
+
+  Scenario: USER users read published material
+    Given a publication with ID "PubId"
```

Comment:
General comment, I find the pattern of `USER users`, `user "theUser"` and `publication "PubId"` to be more confusing than just using `an Anonymous User` and `the Publication`with capital letter to show that they are entities in the scenario.